templates/workspace_services/guacamole/user_resources/guacamole-azure-windowsvm/template_schema.json

```diff
@@ -16,6 +16,22 @@
           "Windows 10",
           "Server 2019 Data Science VM"
         ]
+      },
+      "shared_storage_access": {
+        "$id": "#/properties/shared_storage_access",
+        "type": "string",
+        "title": "Shared Storage Access",
+        "description": "true/false on whether to set-up shared storage access",
+        "enum": [
+          "true",
+          "false"
+        ]
+      },
+      "shared_storage_name": {
```

Comment:
Same as above, can we have a default?

Reply:
Resolving as linked to review on guacamole-azure-linuxvm/template_schema.json